modules/optimizing-mtu-networking.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/optimizing-networking.adoc
+
+[id="optimizing-mtu_{context}"]
+= Optimizing the MTU for your network
+
+There are two important maximum transmission units (MTUs): the network interface card (NIC) MTU and the cluster network MTU.
+
+The NIC MTU is only configured at the time of {product-title} installation. The MTU must be less than or equal to the maximum supported value of the NIC of your network. If you are optimizing for throughput, choose the largest possible value. If you are optimizing for lowest latency, choose a lower value.
+
+The SDN overlay's MTU must be less than the NIC MTU by 50 bytes at a minimum. This accounts for the SDN overlay header. So, on a normal ethernet network, set this to `1450`. On a jumbo frame ethernet network, set this to `8950`.
```

Comment:
And I believe its minus 100 bytes for OVN. We can also include it as reference for OVN as geneve/ovn is already being discussed in this doc. wdyt

Reply:
@anuragthehatter I applied your suggestion. Can you please review the latest change? Thanks!

Reply:
@ahardin-rh Thank you for applying the change. I guess we can put line ```For OVN and Geneve, the MTU must be less than the NIC MTU by 100 bytes at a minimum.``` after we finish sdn discussion.  I means post  ```On a jumbo frame ethernet network, set this to `8950`.```. Currently seems like we are sandwiching ovn details in btw continued SDN dicussion. Will leave up to you to decide. Thank you. Rest LGTM

Reply:
Thanks! I moved the OVN discussion to the right spot.

Reply:
/LGTM